common-test/src/main/java/io/strimzi/test/StrimziRunner.java

```diff
@@ -445,6 +445,11 @@ private Statement withClusterOperator(Annotatable element,
                             String value = envVar.get("value").textValue();
                             ((ObjectNode) envVar).put("value", TestUtils.changeOrgAndTag(value, dockerOrg, dockerTag));
                         }
+                        // Set log level
+                        String logLevel = System.getenv().getOrDefault("ST_STRIMZI_LOG_LEVEL", "INFO");
```

Comment:
You use the variable `logLevel` only inside the `if ... else ...` block ... so why don't you move it into the block instead of querying the environment variables even when you don't need it?

Reply:
Yes, you are right. TBH before this implementation I have something like 
`String logLevel = System.getenv().get("ST_STRIMZI_LOG_LEVEL"`
`if (varName.matches("STRIMZI_LOG_LEVEL") && logLevel) {`
`((ObjectNode) envVar).put("value", logLevel);`
`}`
but then I decided to use method `getOrDefault()` and forgot to move var in block. Thanks for review.